### DISCUSSION TITLE
I wish GitHub mobile app received a push notification every time I am asked to confirm my login with a two digit code. If I had this feature, I could quickly tap on the notification, insert the code, and be done.

### DISCUSSION THREAD

**Discussion by exalted** [community/community#77403] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.140.0 (116339321)
Device: iPhone 11 Pro (iOS 17.1.1)
</details>

**Comment by HenSquared** [comment]
Hi, 

Are you able to receive other push notifications from GitHub, like direct mentions for example? 

If not, please go to your OS  settings, scroll down to app section until you find GitHub 
Select notifications
Toggle 'allow notifications' to on. 

If you're still not receiving notifications, there is probably a bug here. 
Thanks for the feedback!

**Comment by exalted** [comment]
Hey, @HenSquared . Thanks for answering. However, GitHub mobile app doesn't seem to register for notification on my end:

![IMG_4353](https://github.com/community/community/assets/314398/fb0911a4-6cba-424a-9747-f4a51e282095)

**Comment by HenSquared** [comment, marked as answer]
Ah okay. Sorry about that. 

Let's enable notifications in GitHub settings.

Launch the GitHub app.
Go to "Profile" tab
Select "Settings (gear)" icon in top right
Select notifications. 

Tap 'enable notifications'

The OS will then ask permission for GitHub to send you notifications. Choose 'allow'

Let me know if that fixes it. Thanks :)

**Reply by exalted** [reply]
LOL! Thanks. I thought I looked everywhere in the app for something like that, but couldn't fine. Clearly, I wasn't paying enough attention!

Now that notifications are enabled, I received a notification when testing 2-digit login. All good, thank you!!

**Reply by HenSquared** [reply]
Awesome !Glad it works now.
